diff --git a/node_modules/pdf-parse/index.js b/node_modules/pdf-parse/index.js
index e9fc367..47fcd1e 100644
--- a/node_modules/pdf-parse/index.js
+++ b/node_modules/pdf-parse/index.js
@@ -3,7 +3,7 @@ const Pdf = require('./lib/pdf-parse.js');
 
 module.exports = Pdf;
 
-let isDebugMode = !module.parent; 
+let isDebugMode = false; 
 
 //process.env.AUTO_KENT_DEBUG
 
diff --git a/node_modules/pdf-parse/lib/pdf.js/v1.10.100/build/pdf.js b/node_modules/pdf-parse/lib/pdf.js/v1.10.100/build/pdf.js
index d355798..846fc73 100644
--- a/node_modules/pdf-parse/lib/pdf.js/v1.10.100/build/pdf.js
+++ b/node_modules/pdf-parse/lib/pdf.js/v1.10.100/build/pdf.js
@@ -6993,7 +6993,7 @@ var SVGGraphics = function SVGGraphics() {
         if (parseInt(process.versions.node) >= 8) {
           input = literals;
         } else {
-          input = new Buffer(literals);
+          input = Buffer.from(literals);
         }
         var output = require('zlib').deflateSync(input, { level: 9 });
         return output instanceof Uint8Array ? output : new Uint8Array(output);
diff --git a/node_modules/pdf-parse/lib/pdf.js/v1.10.88/build/pdf.js b/node_modules/pdf-parse/lib/pdf.js/v1.10.88/build/pdf.js
index 3eda0ca..9d65829 100644
--- a/node_modules/pdf-parse/lib/pdf.js/v1.10.88/build/pdf.js
+++ b/node_modules/pdf-parse/lib/pdf.js/v1.10.88/build/pdf.js
@@ -7118,7 +7118,7 @@ var SVGGraphics = function SVGGraphics() {
         if (parseInt(process.versions.node) >= 8) {
           input = literals;
         } else {
-          input = new Buffer(literals);
+          input = Buffer.from(literals);
         }
         var output = require('zlib').deflateSync(input, { level: 9 });
         return output instanceof Uint8Array ? output : new Uint8Array(output);
diff --git a/node_modules/pdf-parse/lib/pdf.js/v1.9.426/build/pdf.js b/node_modules/pdf-parse/lib/pdf.js/v1.9.426/build/pdf.js
index be0ba7d..b8a5145 100644
--- a/node_modules/pdf-parse/lib/pdf.js/v1.9.426/build/pdf.js
+++ b/node_modules/pdf-parse/lib/pdf.js/v1.9.426/build/pdf.js
@@ -4096,7 +4096,7 @@ var SVGGraphics = function SVGGraphics() {
         if (parseInt(process.versions.node) >= 8) {
           input = literals;
         } else {
-          input = new Buffer(literals);
+          input = Buffer.from(literals);
         }
         var output = require('zlib').deflateSync(input, { level: 9 });
         return output instanceof Uint8Array ? output : new Uint8Array(output);
diff --git a/node_modules/pdf-parse/lib/pdf.js/v2.0.550/build/pdf.js b/node_modules/pdf-parse/lib/pdf.js/v2.0.550/build/pdf.js
index ff861a4..cf9df24 100644
--- a/node_modules/pdf-parse/lib/pdf.js/v2.0.550/build/pdf.js
+++ b/node_modules/pdf-parse/lib/pdf.js/v2.0.550/build/pdf.js
@@ -15213,7 +15213,7 @@ var SVGGraphics = function SVGGraphics() {
         if (parseInt(process.versions.node) >= 8) {
           input = literals;
         } else {
-          input = new Buffer(literals);
+          input = Buffer.from(literals);
         }
         var output = require('zlib').deflateSync(input, { level: 9 });
         return output instanceof Uint8Array ? output : new Uint8Array(output);
